BetterMD/elements/symbol.py

```
@@ -20,20 +21,28 @@ class Symbol:
     html_written_props = ""
 
-    def __init__(self, styles:'dict[str,str]'={}, classes:'list[str]'=[], dom:'bool'=True, inner:'list[Symbol]'=[], **props):
+    collection = Collection()
+    html_parser = HTMLParser()
+    md_parser = MDParser()
+
+    def __init_subclass__(cls, **kwargs) -> None:
+        cls.collection.add_symbols(cls)
+        super().__init_subclass__(**kwargs)
+
+    def __init__(self, styles:'dict[str,str]'={}, classes:'list[str]'=[], inner:'list[Symbol]'=[], **props):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)